tools/aqrdev

```diff
@@ -852,8 +852,14 @@ def cmd_stop(ctx: AppCtx, name: str) -> None:
 @app.command("shell")
 @click.argument("name", required=True, type=str)
 @click.option("-n", "--node", required=False, type=str)
+@click.option("-c", "--command", required=False, type=str)
```

Comment:
(nit) We should pass in a help string

Reply:
Agreed. This should be done for pretty much all of the commands :\